source/wp-content/themes/wporg-news-2021/sass/blocks/_post-content.scss

```
@@ -0,0 +1,17 @@
+@include break-medium {
+	.single .wp-block-post-content > * {
```

```suggestion
	body.single .wp-block-post-content > * {
```

otherwise it's not clear which `.single` it's referring to

---

🤔 Where else is `single` used in a different context?

---

I can't think of one off the top of my head, but it seems common enough that a plugin etc could use it.

The main benefit IMO is the clarity, so someone isn't left asking "where does that come from? i don't see it anywhere in the theme...", and then have to dig around before they remember the body classes.

Not a big deal though